src/classes/GiftBatchService.cls

```diff
@@ -34,9 +34,36 @@
 */
 public with sharing class GiftBatchService {
 
+    public static final List<String> PAYMENT_FIELDS = new List<String>{
```

Comment:
(NON-BLOCKING) I'm struggling with the name "PAYMENT FIELDS" but not coming up with a better option - but there's maybe a distinction between fields that map to payment but aren't sent to elevate, which would fall under the category of "payment fields" but wouldn't belong in this actual list. "PAYMENT_INTEGRATION_FIELDS"? 

This is non-blocking but if we can think of a better way to represent the concept here that would be tops.

Reply:
I hear what you're saying and at the same time, couldn't think of a better name myself - so, I went ahead and updated the list to use your suggested name of `PAYMENT_INTEGRATION_FIELDS`.